alchemist-web-renderer/gradle.properties

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (C) 2010-2022, Danilo Pianini and contributors
+# listed, for each module, in the respective subproject's build.gradle.kts file.
+#
+# This file is part of Alchemist, and is distributed under the terms of the
+# GNU General Public License, with a linking exception,
+# as described in the file LICENSE in the Alchemist distribution's top directory.
+#
+
+group = it.unibo.alchemist
+artifactId = alchemist-web-renderer
+projectLongName = Alchemist Web Renderer
+projectDescription = Alchemist module that starts a web server and allows to visualize an environment on a web browser.
+
+kotlin.code.style=official
+kotlin.mpp.stability.nowarn=true
+org.gradle.jvmargs=-XX:MaxMetaspaceSize=2g -Xmx2g
```

Comment:
Do you really need a 2g metaspace?

Reply:
I thought the configuration was present in a suggested fullstack-jvm-js multiplatform project I found in the documentation but I was wrong. I probably used this when i tried to make some jvm to js conversion in the first phase of the project and forgot it there. I removed it as the project perfectly works without it.